docs/registry/_index.md

```diff
@@ -26,111 +25,26 @@ aliases:
 menuindent: true
 ---
 
-The [Viam registry](https://app.viam.com/registry) is the central place where you can browse:
+<br>
```

Comment:
cute idea. does unfortunately look a bit wonky. Let's take it out for now and we can worry about making it pretty later.

The title of this page still needs to change though

Reply:
Okay will remove.

I changed the title of the other page but what did you want the title of this page to be? Sorry if I missed something but I cannot find that comment.